.env.example

```diff
@@ -1,9 +1,9 @@
 NODE_ENV=
 PORT=
 
-DATABASE_URL=
-DATABASE_TEST_URL=
-DATABASE_DIALECT=
+DB_USERNAME=your_db_username
+DB_PASS=your_db_password
```

Comment:
- Where does the prod config file live and is there a way for us to avoid having creds as part of our code? I am not too familiar with mySQL deployment best practices. 
- Are we configuring the mySQL db to only communicate with the app ?
- Do both App and mySQL live in the same container ?

Reply:
- deployment CI/CD not setup yet, but config are stored in a `.env` which is not pushed to VCS
- for now yes, when there is an implementation to containerize the app it can be connnected to other services like `adminer`, `pgAdmin`, or `seperate analytics codebase` etc.
- Containerization not implemented yet

Reply:
Throw in a TODO comment here stating these have to be removed on containerization. This will serve as a constant reminder.

Reply:
Actually, no they would not be removed on containerization, they would still be used from the environment. Please see the readme https://github.com/BanTheBad/middleware/blob/develop/README.md for info on how the env variables are being used, this file is a sample.